src/domain/vesoljci.py

```diff
@@ -2,7 +2,16 @@
 
 from src.domain.objekt import Objekt
 
-
+# TODO: TUKAJ BI MOGOČE DODAL INITIALIZACIJO VELIKOSTI VESOLJCA ?
 @dataclass
 class Vesoljci(Objekt):
-    pass
\ No newline at end of file
+    velikost_x: float
+    velikost_y: float
+    hitrost: float
+    smer: str
```

Comment:
Tukaj ne potrebuješ lastnosti smeri saj ti bo hitrost_x in hitrost_y povedalo v katero smer se marsovc premika.